person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..255)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return true if of_age? || @parent_permission
+
+    false
+  end
```

Comment:
- [OPTIONAL ] To make sure that the can_use_services? looks more straightforward and simple I would recommend you refactor it to this :point_down: 

```
  def can_use_services?
      of_age? || @parent_permission
  end

```